L1Trigger/L1TMuon/plugins/L1TMuonProducer.cc

```diff
@@ -46,23 +47,27 @@
 
 #include "CondFormats/L1TObjects/interface/L1TMuonGlobalParams.h"
 #include "CondFormats/DataRecord/interface/L1TMuonGlobalParamsRcd.h"
+#include "CondFormats/DataRecord/interface/L1TMuonGlobalParamsO2ORcd.h"
 #include "L1Trigger/L1TMuon/interface/L1TMuonGlobalParamsHelper.h"
+#include "L1Trigger/L1TMuon/interface/L1TMuonGlobalParams_PUBLIC.h"
 
 #include "TMath.h"
 //
 // class declaration
 //
 using namespace l1t;
 
-  class L1TMuonProducer : public edm::stream::EDProducer<> {
+  class L1TMuonProducer : public edm::EDProducer {
```

Comment:
@rekovic why was this moved back to being an EDProducer? Was this a mistake when trying to resolve the conflict?